project/MimaVersionPlugin.scala

```diff
@@ -87,13 +87,24 @@ object MimaVersionPlugin extends AutoPlugin {
 
       val isPre = major.toInt == 0
 
-      val isJvm = virtualAxes.?.value
+      val isScalaJs = virtualAxes.?.value
         .getOrElse(List.empty)
         .contains(
-          VirtualAxis.jvm
+          VirtualAxis.js
         )
 
-      if (sbtPlugin.value || !isJvm) {
+      val isScalaNative = virtualAxes.?.value
+        .getOrElse(List.empty)
+        .contains(
+          VirtualAxis.native
+        )
+
+      val suffix =
+        if (isScalaJs) "_sjs1"
+        else if (isScalaNative) "_native0.4"
```

Comment:
Anyone know of a way to get these values from the build directly? CrossVersion something?

Reply:
Here's what sbt-typelevel does so it doesn't have to play games with suffixes and stuff :)

https://github.com/typelevel/sbt-typelevel/blob/cedbed34d087ac2a0f24112eaa5f128c906deae0/mima/src/main/scala/org/typelevel/sbt/TypelevelMimaPlugin.scala#L88

Reply:
Awesome, thanks! I updated to use that same mechanism.